### DISCUSSION TITLE
Great project!

### DISCUSSION THREAD

**Discussion by ghost** [blue-build/.github#21] [Show and tell]
Like to thank you all for the great efforts to make this happen; look at the result over here ;-) 
https://github.com/thesaigoneer/atomic-kde-saigon

Best regards, Mark
![Screenshot_20240408_175351](https://github.com/blue-build/.github/assets/165976396/02481e92-c7d3-41b3-b001-576b83d960f1)

**Comment by xynydev** [comment]
Thanks for the thanks and your thoughts! Always nicr to get experience reports.
